web/js/map/ui.js

```diff
@@ -41,7 +41,7 @@ export function mapui(models, config) {
   var createLayer;
   var precache = mapPrecacheTile(models, config, cache, self);
 
-  var dataRunner = self.runningdata = new MapRunningData(models);
+  var dataRunner = self.runningdata = new mapRunningData(models);
```

Comment:
@localjo Should this remain Capitalized as it uses a constructor pattern? If we decide against keeping it capitalized, `MapRotate` should probably be lowercase as well.

Reply:
Good catch.